rfcs/0019-olmv1-operator-consolidation.md

```
@@ -23,292 +40,237 @@ Bundles declaring `olm.package.required`, `olm.gvk.required`, or `olm.constraint
-
-Users install Kuadrant directly via Helm:
-
-```bash
-helm install kuadrant kuadrant/kuadrant
-```
-
-The chart deploys all components — kuadrant-operator, Authorino, Limitador, and dns-operator — with configurable values for image references, replicas, resource limits, storage backend, and TLS.
+The kuadrant-operator becomes an umbrella operator that deploys component controllers at runtime. Wrapper CRs (Authorino CR, Limitador CR) are preserved. No breaking changes for users.
 
-### Install on OpenShift (OLM)
+### Before
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)